iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/schema/filter/SchemaFilterFactory.java

```diff
@@ -44,6 +45,10 @@ public static SchemaFilter createViewTypeFilter(ViewType viewType) {
     return new ViewTypeFilter(viewType);
   }
 
+  public static SchemaFilter createTemplateIdFilter(String templateName, boolean isEqual) {
```

Comment:
Please let the name of the method express the exact meaning.
```suggestion
  public static SchemaFilter createTemplateNameFilter(String templateName, boolean isEqual) {
```

Reply:
Fixed.